src/DaPigGuy/PiggyCustomEnchants/enchants/weapons/HallucinationEnchant.php

```diff
@@ -35,14 +35,6 @@ class HallucinationEnchant extends ReactiveEnchantment
     /** @var array */
     public static $hallucinating;
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 30, so I commented this line